ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-intended-config.j2

```diff
@@ -6,16 +6,20 @@
 {% include 'eos/boot.j2' %}
 {# terminal settings #}
 {% include 'eos/terminal.j2' %}
-{# cli prompt #}
-{% include 'eos/prompt.j2' %}
-{# aliases #}
-{% include 'eos/aliases.j2' %}
 {# hardware counters #}
 {% include 'eos/hardware-counters.j2' %}
+{# enable password #}
```

Comment:
please revert sequence changes for anything not related to `ip xxx client`

Reply:
Fixed!